route_guide/route_guide_client.cc

```diff
@@ -263,17 +349,86 @@ class RouteGuideClient {
 int main(int argc, char** argv) {
   // Expect only arg: --db_path=path/to/route_guide_db.json.
   std::string db = routeguide::GetDbFileContent(argc, argv);
-  Client client("localhost:50051", grpc::InsecureChannelCredentials());
-  RouteGuideClient guide(db, &client);
 
+  // Borrowable<T> is data structure which lets us to get borrowed_ptr<T>()
+  // that we can pass like a unique_ptr<T> and can give to some part of the
+  // code a bunch of things and when they are all done with it, when the last
+  // object which points to the data is done with borrowed_ptr, the object
+  // which originally created this pointer can find out about that(using
+  // watch callback which will tell when the object will no be longer
+  // borrowed). More about Borrowable class is in the link below:
+  // https://github.com/3rdparty/stout-borrowed-ptr/blob/master/stout/borrowable.h
+  Borrowable<CompletionPool> pool;
+  Client client("localhost:50051", grpc::InsecureChannelCredentials(),
+                pool.Borrow());
+  RouteGuideClient guide(db, &client);
+  // -------------------------------------------------------------------------
+  // -------------------------------------------------------------------------
   std::cout << "-------------- GetFeature --------------" << std::endl;
-  guide.GetFeature();
+
+  // Unary stuff is working
```

Comment:
We don't need to have comments saying that things work. That's the expected case. 😉 

If you want to remind people what it is we're demonstrating, then you can rephrase as...
```suggestion
  // Demonstrate unary (single-request, single-response) RPCs:
```